### ISSUE TITLE
Override all WordPress content files when running first provisioning

### ISSUE THREAD

**Issue by youthkee** [vccw-team/vccw#256]
I’m using VCCW v3 with Git like as [repository-example-of-wp](https://github.com/vccw-team/repository-example-of-wp).

But I clone the repository with VCCW in another PC and provisioning it, There seems to be updated all files under `/wordpress/` directory in spite of no differences than before.

This event doesn’t occur with VCCW v2.

Is it possible to override only changed files when running first provisioning?

Operating System: OSX 10.10.5
Vagrant Version: 1.9.6
Virtual Box Version: 5.1.22

## Just cloning the repository

![repository-example-of-wp01](https://user-images.githubusercontent.com/319783/27812159-b07e7970-60a7-11e7-875d-8474076a15fd.png)

## After running first provisioning

![repository-example-of-wp02](https://user-images.githubusercontent.com/319783/27812162-b4aa1428-60a7-11e7-9d23-71180220aab6.png)

**Comment by miya0001** [comment]
Are themes and plugins updated or deleted?
I think WordPress core should always be updated.

**Comment by youthkee** [comment]
In this repository, I added only an original theme (not default themes) and media files in `/uploads/` directory.
These files also have been overridden(or change time stamps) when running provisioning, in spite of they are not WordPress core files.
https://github.com/littlebirdjp/vccw-env-for-theme-build

**Comment by miya0001** [comment]
Could you give me a output of the `ls -al` or `git status` or so.
I am sorry I can't understand your problem and I don't have enough time to `vagrant up` your project.

I want to know what was changed. timestamp of the file was changed? file contents was modifiled? file was deleted? or file mode was changed???

**Comment by youthkee** [comment]
I’m sorry for not telling details.

I tried to  `git status` and `ls -al -R` before and after provisioning both on [my repository](https://github.com/littlebirdjp/vccw-env-for-theme-build).
As the result, all existing files under `/wordpress/` directory have changed git status to `modified`.
And in comparison with `la -al -R` command, existing files under `/wordpress/` directory have changed permissions from `-rw-r—r—`(644) to `-rwxr-xr-x`(755).
I also tried to compare those files by a diff tool, but there are no changes on source code or as binary data between them.

So, the problem is that if there are any files under `/wordpress/` directory previously, those files have been changed permissions to 755 when running provisioning.

And just now, I noticed that this problem would be solved by using `git config core.filemode false` in order to ignore permission changes.
It will be OK to close this Issue.
(But is it a proper action for VCCW...?)
